src/sbuf/SBuf.cc

```diff
@@ -70,7 +69,7 @@ SBuf::~SBuf()
 {
     debugs(24, 8, id << " destructed");
     --stats.live;
-    recordSBufSizeAtDestruct(len_);
+    SBufStats::RecordSBufSizeAtDestruct(len_);
```

Comment:
Since this method may throw ( StatHist is created dynamically), should not we protect our destructor? I just wonder whether such problems are considered minor (and we just turn a blind eye to it).

Reply:
> but that (old and widespread) bug is not for _this_ PR to fix IMO. 

I agree. Non-regression is a good enough criteria for accepting for me